doc/nrf/app_dev/device_guides/nrf54l/ecies_x25519.rst

```diff
@@ -0,0 +1,80 @@
+.. _ug_nrf54l_ecies_x25519:
```

Comment:
In addition to VS code instructions, please include the reference to this page in the following file and line:
https://github.com/nrfconnect/sdk-nrf/blob/24d63eb67e72847e6b7d2ef19502857aab3219c8/doc/nrf/app_dev/device_guides/nrf54l/dfu_config.rst?plain=1#L137

I.e.:

``For detailed information on ECIES-X25519 support, refer to the :ref:`ug_nrf54l_ecies_x25519` documentation page.``